sympy/core/relational.py

```diff
@@ -285,6 +291,13 @@ def equals(self, other, failing_expression=False):
                     return right
                 return left
 
+    # XXX: This is copied from Expr. Maybe it should be defined on Basic or in
+    # a mixin instead.
+    def simplify(self, **kwargs):
+        """See the simplify function in sympy.simplify"""
+        from sympy.simplify import simplify
+        return simplify(self, **kwargs)
```

Comment:
How about making this return something like `self.func(self.lhs.simplify(**kwargs), self.rhs.simplify(**kwargs))`.

Reply:
That wouldn't call `Relational._eval_simplify` above. The first step of `_eval_simplify` is to simplify both lhs and rhs but then it does a load more to try and evaluate the relation to true/false.

Reply:
We currently have things like
```julia
In [2]: Ge(x - y, x + y).simplify()                                                                                               
Out[2]: y ≤ 0
```

Reply:
I didn't notice `_eval_simplify`. The code in `simplify` looks like doing nothing much on relations. Then clearly this method should be preserved. Or maybe it should rather be moved to `Basic`. I don't think that any harm would result.